pkg/astnormalization/astnormalization.go

```diff
@@ -180,7 +180,12 @@ func (o *OperationNormalizer) setupOperationWalkers() {
 	if o.options.removeUnusedVariables {
 		deleteUnusedVariables(&other)
 	}
-	o.operationWalkers = append(o.operationWalkers, &fragmentInline, &extractVariablesWalker, &other)
+
+	inputDefaultInjectorWalker := astvisitor.NewWalker(48)
```

Comment:
Why don't you just use the `other` visitor?

Reply:
oh thanks for pointing that out, i'll need to write a bit more tests with other visitor edge cases to check this though